Docs: Clarify running `make install` when changing dependencies (#7944)
plone/volto: docs/source/contributing/developing-core.md

```

> (developing-core-when-changing-dependencies-label)=

Reviewer: ```suggestion
(developing-core-dependency-updates-label)=
```

> ### When you change a dependency

Reviewer: Either the reader or another developer through a merged pull request updates dependencies. This heading addresses both actors, instead of only the reader.

```suggestion
### Dependency updates
```

> Whenever you add, remove, or update a dependency in {file}`package.json` (in the repository root or in any workspace under {file}`packages/`), you must:

Reviewer: ```suggestion
Whenever you add, remove, or update a dependency in {file}`package.json`, either in the repository root or in any workspace under {file}`packages/`, you must:
```

1.  Run `make install` from the repository root.

> 2.  Commit the updated {file}`pnpm-lock.yaml` file.

Reviewer: ```suggestion
2.  Commit the updated {file}`pnpm-lock.yaml` lockfile.
```

> If you skip this, the lockfile will be out of date and tests may fail locally and in continuous integration.

Reviewer: ```suggestion
If you skip these steps, the lockfile will be out of sync.
Tests may fail both locally and, after you push the commit, in continuous integration.
```

> New contributors should run `make install` after pulling changes that touch dependencies or the lockfile.

Reviewer: The audience is anyone who contributes to core, not just new one contributors.

It should also be a new paragraph so that it stands out and doesn't get lost in the previous context.

```suggestion

Additionally, after you pull changes from upstream that touch either dependencies or the lockfile, you should run `make install`.
```
